src/components/Products.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import '../styles/Products.css';
+import ProdcutsCard from "./Cards";
```

Comment:
This says `./Cards` , probably this should have been `./ProductsCard`

Reply:
Damn that's embarrassing :')
Quick fix incoming